.travis.yml

```diff
@@ -24,8 +24,8 @@ jobs:
       env: ARCH=riscv LLVM_IAS=1
     - name: "ARCH=s390 BOOT=false"
       env: ARCH=s390 BOOT=false
-    - name: "ARCH=x86 LD=ld.lld"
-      env: ARCH=x86 LD=ld.lld
+    - name: "ARCH=x86 LLVM=1"
```

Comment:
Let's get #302 merged then switch over as much as possible to `LLVM=1` + GNU variants as we need them.

Reply:
will do